internal/rbd/mirror.go

```diff
@@ -69,6 +69,24 @@ func (ri *rbdImage) getImageMirroringInfo() (*librbd.MirrorImageInfo, error) {
 	return info, nil
 }
 
+// checkHealthyPrimary checks if the image is in up+stopped state.
+func (ri *rbdImage) checkHealthyPrimary() error {
+	mirrorStatus, err := ri.getImageMirroringStatus()
+	if err != nil {
+		return fmt.Errorf("failed to get mirroring info of %q with error: %w", ri, err)
```

Comment:
`fmt.Errorf("failed to get mirroring info of %q: %w", ri, err)` , `with error` is not required.